pages/api/user/[id].ts

```diff
@@ -14,19 +15,41 @@ export default async function handle(
 ): Promise<void> {
     if (req.method == "GET") {
         // Obtain user id
-        const { id } = req.query;
+        const { userId } = req.query;
```

Comment:
This won't work I believe. You have to have it as the same variable in the file name `[id].ts`. Did you guys test this?